presto-main/src/test/java/com/facebook/presto/resourcemanager/TestResourceManagerClusterStateProvider.java

```diff
@@ -503,7 +503,7 @@ public void testWorkerMemoryInfo()
         assertWorkerMemoryInfo(provider, 2);
 
         // Expire nodes
-        Thread.sleep(SECONDS.toMillis(5));
+        Thread.sleep(SECONDS.toMillis(15));
```

Comment:
Should we add some comments on why we need to wait for that long ? Is it too hard to use something better than sleep ?

Reply:
+1

Reply:
Added comment to explain the reason for the sleep. Can't think of anything better than sleep here given we want the expiration time to pass. Please let me know if there is something better can be done here.